lib/handler.js

```diff
@@ -1,6 +1,8 @@
-const _ = require('lodash');
-const aeclient = require('aws-elasticsearch-client');
-const AWS = require('aws-sdk');
+const has = require('lodash/has');
+const pick = require('lodash/pick');
+const keys = require('lodash/keys');
+const defaults = require('lodash/defaults');
+const DynamoDB = require('aws-sdk/clients/dynamodb');
```

Comment:
`const { DynamoDB } = require('aws-sdk');` ?

Reply:
That would load the entire `aws-sdk` library with all of its services. This way, we only load the `dynamodb` service. I was hoping to *only* load the converter, but the SDK isn't set up for that--at least, I couldn't get it to work right.

Reply:
yeah, I'm just always worried about relying on the implementation details of the libraries.

Reply:
If it helps, this manner of loading portions of the SDK is documented and supported - see https://www.npmjs.com/package/aws-sdk. They won't break it in a minor version update.